autoload/black.vim

```diff
@@ -3,8 +3,13 @@ import collections
 import os
 import sys
 import vim
-from distutils.util import strtobool
 
+def strtobool(text):
+    if text.lower() in ['y', 'yes', 't', 'true' 'on', '1']:
```

Comment:
We seem to be using two-space indents in this file for some reason, so let's stay consistent with that.

Also, please use an f-string on line 12.